server/package.json

```diff
@@ -51,12 +51,6 @@
     "open": "0.0.5",
     "pa11y": "^4.2.0"
   },
-  "babel": {
-    "presets": [
-      "es2015",
-      "stage-3"
```

Comment:
Nope, as we're now using [babel-env](https://github.com/babel/babel-preset-env).